config/routes.rb

```diff
@@ -13,5 +13,8 @@
     resources :likes, only: [:create, :destroy]
   end
   
+  get '/api/posts', to: 'api#show_posts'
+  get '/api/posts/:post_id/comments', to: 'api#show_comments'
+  post '/api/posts/comment', to: 'api#add_comment'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 end
```

Comment:
- [ ] Kindly provide endpoints for authenticating the user.
-  Login endpoint - **POST :** `/login`
- Registration endpoint - **POST :** `/register` or `/signup`